activestorage/lib/active_storage/attached/model.rb

```diff
@@ -88,7 +101,17 @@ def #{name}=(attachable)
       #
       # If the +:dependent+ option isn't set, all the attachments will be purged
       # (i.e. destroyed) whenever the record is destroyed.
-      def has_many_attached(name, dependent: :purge_later)
+      #
+      # If you need the attachment to use a service, which differs from the globally configured one
+      # you should pass +:service+ option, for instance:
```

Comment:
`If you need the attachment to use a service which differs from the globally configured one, pass the +:service+ option. For instance:`